hack/generator/pkg/armconversion/convert_from_arm_function_builder.go

```diff
@@ -86,7 +86,7 @@ func (builder *convertFromARMBuilder) functionDeclaration() *dst.FuncDecl {
 	fn.AddComments("populates a Kubernetes CRD object from an Azure ARM object")
 	fn.AddParameter(
 		builder.idFactory.CreateIdentifier(astmodel.OwnerProperty, astmodel.NotExported),
-		astbuilder.Selector(dst.NewIdent(astmodel.GenRuntimePackageName), "KnownResourceReference"))
+		astbuilder.Selector(dst.NewIdent(astmodel.GenRuntimeReference.PackageName()), "KnownResourceReference"))
```

Comment:
This comment made me realize there was a better way to do this:
`astmodel.KnownResourceReferenceType.AsType(codeGenerationContext)`, which sets this under the hood and is cleaner because it doesn't duplicate the `"KnownResourceReference"` text. I've moved to use that which I think fixes this issue.